lua/compe/completion.lua

```diff
@@ -231,7 +231,9 @@ end
 Completion._show = function(start_offset, items)
```

Comment:
I think we should wrap all `Completion._show` codes by `vim.schedule`.

Reply:
Yeah! This would make the behavior more predictable in terms of execution.